src/client/store/project.js

```diff
@@ -825,7 +827,6 @@ export const getters = {
           }
           return obj
         }, {})
-      return { ...kpiValues, returnTime: returnTime + 1 }
```

Comment:
something goes wrong here I think, this was added by @bob-voorhoede in an other ticket, maybe a merge issue?